packages/server/src/api/muster/muster.controller.it.ts

```diff
@@ -20,25 +21,32 @@ describe(`Muster Controller`, () => {
     org1 = seedData.find(data => data.org.indexPrefix === 'testgroup1').org.id;
   });
 
-  describe(`${basePath}/:orgId/:unitName/complianceByDate : get`, () => {
+  describe(`${basePath}/:orgId/roster/complianceByDateRange : get`, () => {
     it(`should return full compliance with multiple active configs`, async () => {
-      const res = await req.get(`/${org1}/${unit5}/complianceByDate?isoDate=2020-01-02`);
+      const res = await req.get(`/${org1}/roster/complianceByDateRange?fromDate=2020-01-01&toDate=2020-01-03${pageParams}`);
```

Comment:
I think we can pass these query strings through using the req.get `config` argument like this. It should let you leverage type checking when building the query, and it'll make things a bit more readable.

```typescript
const query: MusterComplianceByDateRangeQuery = {
  fromDate: '2020-01-01',
  toDate: '2020-01-03',
}

const res = await req.get(`/${org1}/roster/complianceByDateRange`, {
  params: query,
});
```

One interesting thing to note, that I only recently discovered, is that type checking is more strict when using `: Type` vs `as Type`, which is why I define the query here before the request happens.

I hate how Axios takes a `params` property for the query string, but it'll get the job done anyways.